flow/process.go

```diff
@@ -238,16 +243,52 @@ func getApplicationByImage(image string) (*Application, error) {
 	return nil, errors.New("No application found for image " + image)
 }
 
-func generateBody(app Application, manifest Manifest, version string, oldVersions []string) string {
+func generateBody(ctx context.Context, client *github.Client, app *Application, manifest Manifest, version string, oldVersions []string) string {
 	var body string
+
 	if !manifest.HideSourceReleaseDesc {
-		body += "## Release\n"
+		body += "# Release\n"
 		body += fmt.Sprintf("https://github.com/%s/%s/releases/tag/%s\n", app.SourceOwner, app.SourceName, version)
 		body += "\n"
 
-		body += "### Diff from last release\n"
+		body += "## Changes\n\n"
 		for _, oldVersion := range oldVersions {
-			body += fmt.Sprintf("https://github.com/%s/%s/compare/%s...%s\n", app.SourceOwner, app.SourceName, oldVersion, version)
+			body += fmt.Sprintf("https://github.com/%s/%s/compare/%s...%s\n\n", app.SourceOwner, app.SourceName, oldVersion, version)
+			if !manifest.HideSourceReleasePullRequests {
```

Comment:
I introduced a flag because calling GitHub API may fails and may want to disable this feature.